### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Desomph**
Fixed request and response, JSON is now properly camel cased.
[FromQuery] parameter is naturally case-insensitive so NSwag UI displays params as property names (upper case).

### CODE REVIEW

File: Exebite.API/Extensions/NSwagExtension.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.DependencyInjection;
```
Comment (stanoje) [inline Exebite.API/Extensions/NSwagExtension.cs]: I like this, moving complexity separate file/component

File: Exebite.API/Startup.cs [hunk 2]
```diff
@@ -84,10 +83,9 @@ public void ConfigureServices(IServiceCollection services)
 
             .AddDataAccessServices()
             .AddCommonServices();
-            services.Configure<IISOptions>(x =>
-            {
-                x.ForwardClientCertificate = false;
-            });
+
+            services.Configure<IISOptions>(x => x.ForwardClientCertificate = false);
```
Comment (stanoje) [inline Exebite.API/Startup.cs]: Try to keep the previous formatting or create a separate pull request with just formatting.

File: Exebite.API/Extensions/NSwagExtension.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {
+        /// <summary>
+        /// Add NSwag settings in regards to CamelCase (de)serialization of objects.
+        /// </summary>
+        /// <returns>IMvcBuilder</returns>
+        public static IMvcBuilder AddNSwagSettings(this IMvcBuilder builder) =>
+            builder.AddJsonOptions(opt =>
+                {
+                    opt.SerializerSettings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+                    opt.SerializerSettings.Converters.Add(new StringEnumConverter());
```
Comment (stanoje) [inline Exebite.API/Extensions/NSwagExtension.cs]: This would change the behavior of REST API (it will generate different type int => string) and break clients functionality. 

As no one is consuming this API's it will not break anything, but be aware of changes like this as this is a system/application wide change

Comment (Desomph) [reply Exebite.API/Extensions/NSwagExtension.cs]: Thanks for letting me know, I'll remove the Enum converter!

File: Exebite.API/Startup.cs (hunk 2 above)
Comment (Desomph) [reply Exebite.API/Startup.cs]: Restored previous formatting, will make a separate pull request if needed.

**Comment by Desomph** [comment]
Made the changes requested, commit ready for review.

**Comment by stanoje** [comment]
Please complete the pull request
